javascript-ajax/users.js

```diff
@@ -0,0 +1,19 @@
+const userList = document.querySelector('#user-list');
+
+const xhr = new XMLHttpRequest();
+
+xhr.open('GET', ' https://jsonplaceholder.typicode.com/users' + name);
```

Comment:
I don't see any data here when I test this URL^  "https://jsonplaceholder.typicode.com/usersname

The correct URL is: `https://jsonplaceholder.typicode.com/users`